src/app/component/burger/build_controls.cljs

```diff
@@ -8,29 +8,23 @@
   (action [{:keys [state]}]
     (let [id (:id props)
           price (:price props)]
-      ;(js/console.log id price (get-in @state [:order/id 1 :order/total-price]))
       (swap! state update-in [:order/id 1 :order/ingredients id :count] inc)
       (swap! state update-in [:order/id 1 :order/total-price] + price))))
 
 (defmutation remove-ingredient [props]
   (action [{:keys [state]}]
      (let [id (:id props)
            price (:price props)]
-       ;(js/console.log id price (get-in @state [:order/id 1 :order/total-price]))
        (swap! state update-in [:order/id 1 :order/ingredients id :count] dec)
        (swap! state update-in [:order/id 1 :order/total-price] - price))))
 
-
 (defmutation toggle-modal [_]
   (action [{:keys [state]}]
-    (let [modal? (get-in @state [:ui/id 1 :ui/modal?])]
-      (js/console.log "Toggle Modal: "  modal?)
-      (swap! state assoc-in [:ui/id 1 :ui/modal?] (not modal?)))))
-
+    (let [modal? (get-in @state [:singleton :app.ui/modal :ui/modal])]
+      (swap! state assoc-in [:singleton :app.ui/modal :ui/modal] (not modal?)))))
```

Comment:
Je kunt hier beter `update-in` gebruiken

Reply:
ja, klopt. gedaaan